sound/soc/sof/intel/hda-ipc.c

```diff
@@ -148,6 +149,33 @@ irqreturn_t hda_dsp_ipc_irq_thread(int irq, void *context)
 	hipci = snd_sof_dsp_read(sdev, HDA_DSP_BAR, HDA_DSP_REG_HIPCI);
 	hipcte = snd_sof_dsp_read(sdev, HDA_DSP_BAR, HDA_DSP_REG_HIPCTE);
 
+	/* is this a new message from DSP */
```

Comment:
@lyakh I can only envision this possibility if the FW panic happens while the host is processing the reply in which case we will go on and process the panic right after. So not sure if this change is absolutely required.